src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -31,6 +31,8 @@
                                          weather_data,
                                          curve_number,
                                          sediment_phosphorus,
+                                         groundwater_nitrogen_conc,
+                                         sediment_delivery_ratio
```

Comment:
Always add a comma at the end of lists in Python, even in the last element. That way if we add more elements after this, this line won't change.